resources/config.edn

```diff
@@ -25,8 +25,8 @@
   ; For details, see https://github.com/rhuss/jolokia/blob/master/agent/jvm/src/main/resources/default-jolokia-agent.properties
   ; Note: All keys and values MUST be strings - this is a Jolokia requirement
   :jolokia-config {
-    "host"  #or [#env JOLOKIA_HOST  "localhost"]
-    "port"  #or [#env JOLOKIA_PORT  "8778"]
+    "host"  #or [#env JOLOKIA_HOST "*"]
```

Comment:
This should be configured in the OpenShift environment, not in the default `config.edn` file, since opening up Jolokia in this way is potentially a massive security hole (so that should not be the default).

Reply:
Also, need to preserve horizontal alignment here too.